package.json

```diff
@@ -8,12 +8,11 @@
     "test": "npx jest",
     "start": "webpack-dev-server --mode development --open",
     "build": "npm run clean; npx webpack --mode development",
-    "build:prod": "npm run clean; npm run lint && npx webpack --mode production && npx tsc && npm run copy-json && npm run docs",
+    "build:prod": "npm run clean; npm run lint && npx webpack --mode production && npm run docs",
     "lint": "npm run lint-ts && npm run lint-sass",
     "format": "npx prettier --write *",
     "docs": "npx typedoc",
-    "clean": "rm -rf dist",
-    "copy-json": "mkdir -p dist/js/framework/localization/languages && cp src/ts/localization/languages/*.json dist/js/framework/localization/languages/",
```

Comment:
Where is the `copy-json` now handled? I can't spot the replacement.

Reply:
This is done automatically and directly by the `npx tsc` command (in `webpack.config.js#createJavascriptUiFrameworkFilesWithReplacedPrefix`).